src/Juvix/Compiler/Core/Data/Stripped/InfoTable.hs

```diff
@@ -46,6 +46,7 @@ data ConstructorInfo = ConstructorInfo
     _constructorTag :: Tag,
     _constructorType :: Type,
     _constructorArgNames :: [Maybe Text],
+    _constructorArgsNum :: Int,
```

Comment:
`_constructorArgsNum` is always equal to `length (typeArgs _constructorType)`, I guess? There should be a comment that this field is only a "cache" for this value.